modules/dublincore/src/test/java/org/opencastproject/metadata/dublincore/StaticMetadataServiceDublinCoreImplTest.java

```diff
@@ -43,17 +38,6 @@
 
 public final class StaticMetadataServiceDublinCoreImplTest {
 
-  @Test
-  public void testExtractMetadata() throws Exception {
-    MediaPackage mp = newMediaPackage("/manifest-simple.xml");
-    StaticMetadataServiceDublinCoreImpl ms = newStaticMetadataService();
-    StaticMetadata md = ms.getMetadata(mp);
-    assertEquals("Land and Vegetation: Key players on the Climate Scene",
-            md.getTitles().stream().filter(v -> v.getLanguage().equals(MetadataValues.LANGUAGE_UNDEFINED))
-                    .findFirst().map(MetadataValue::getValue).orElse(""));
-    assertEquals(createDate(2007, 12, 5, 0, 0, 0), md.getCreated().get());
-  }
-
```

Comment:
Sure, if we get rid of `getCreated()` we can get rid of the later assertion, but why get rid of the former assertion as well?